src/main/java/net/sf/jabref/logic/layout/format/LatexToUnicodeFormatter.java

```diff
@@ -31,6 +31,7 @@
 public class LatexToUnicodeFormatter implements LayoutFormatter, Formatter {
 
     private static final Map<String, String> CHARS = HTMLUnicodeConversionMaps.LATEX_UNICODE_CONVERSION_MAP;
+    private static final Map<String, String> ACCENTS = HTMLUnicodeConversionMaps.UNICODE_ESCAPED_ACCENTS;
```

Comment:
@JabRef/developers Does java copies here the whole map or just point to the original one? If the map is copied, then one should inline `CHARS` and `ACCENTS`